src/reggae/dub/interop/dublib.d

```diff
@@ -386,8 +391,9 @@ class InfoGenerator: ProjectGenerator {
         if(settings.platform.platform.canFind("linux"))
             pkg.lflags = "-L--no-as-needed" ~ pkg.lflags;
 
+        auto dflags = buildSettings.dflags.chain(_extraDFlags);
```

Comment:
My first attempt was to use `import reggae.config: options;` and `options.dflags` here. But that was empty for the test case, so I went with the ugly forwarding of these extra `dflags`.